Kickstarter-iOS/ViewModels/AppDelegateViewModelTests.swift

```diff
@@ -10,7 +10,7 @@ import UserNotifications
 @testable import LiveStream
 @testable import Kickstarter_Framework
 @testable import KsApi
-@testable import ReactiveExtensions_TestHelpers
+import ReactiveExtensions_TestHelpers
```

Comment:
We can't use `@testable` to import `ReactiveExtensions_TestHelpers` because it's external to our project now. This appears to not be a problem.

The one thing I had to do on the framework side was to make `TestObserver` public to workaround this, I don't believe that's an issue.